tabcmd/parsers/help_parser.py

```diff
@@ -0,0 +1,17 @@
+import sys
+import argparse
+
+
+class HelpParser:
+    @classmethod
+    def print_help_description(cls):
+        description = \
+            "tabcmd - Tableau Server Command Line Utility 2.0 \n \n" \
```

Comment:
shouldn't argparse generate this for us?

Reply:
Thanks for reviewing. 
I had to override the Tabcmd 2 argparse help message to to mirror the same help message given out in Tabcmd1.